config/v1/types_authentication.go

```diff
@@ -195,74 +195,118 @@ const (
 )
 
 type OIDCProvider struct {
-	// name of the OIDC provider
+	// name configures the unique human-readable identifier
+	// associated with the identity provider.
+	// name is used to distinguish between multiple identity providers.
+	// name has no impact on token validation or authentication mechanics.
+	// name is required.
+	// name must not be an empty string ("").
 	//
 	// +kubebuilder:validation:MinLength=1
 	// +required
 	Name string `json:"name"`
-	// issuer describes atributes of the OIDC token issuer
+
+	// issuer configures how the platform interacts
+	// with the identity provider and how tokens issued
+	// from the identity provider are evaluated by the
+	// Kubernetes API Server.
```

Comment:
This looks like a more suitable comment for the type `TokenIssuer` instead of the field.

Reply:
Comments on top level types do not get included in the generated OpenAPI spec field documentation that is shown to end users for fields of that type. This comment is here so that it appears for the end-user and explains what this field is used for if they run something like `oc explain ...`